js/background.js

```diff
@@ -137,24 +65,38 @@ chrome.runtime.onMessage.addListener(function (message, sender, sendResponse)
 			RemoveTab(message.collectionIndex, message.tabIndex);
 			sendResponse();
 			break;
-		case "toFavorites":
-			AddTabsToFavorites(message.collectionIndex);
-			break;
-		case "share":
-			ShareTabs(message.collectionIndex);
-			break;
 	}
 });
 
+// This function updates the extension's toolbar icon
+function UpdateTheme()
+{
+	var theme = window.matchMedia("(prefers-color-scheme: dark)").matches ? "dark" : "light";
+	var iconStatus = collections.length ? "full" : "empty";
+
+	var basePath = "icons/" + theme + "/" + iconStatus + "/";
+
+	chrome.browserAction.setIcon(
+		{
+			path:
+			{
+				"128": basePath + "128.png",
+				"48": basePath + "48.png",
+				"32": basePath + "32.png",
+				"16": basePath + "16.png"
+			}
+		});
+}
+
 UpdateTheme();
 chrome.windows.onFocusChanged.addListener(UpdateTheme);
-
 chrome.tabs.onUpdated.addListener(UpdateTheme);
```

Comment:
As I think I mentioned before, we can't listen for theme changes by ourselves in background scripts.

So the trick with this code is that some tabs update on theme change. For example, github detects theme change and updates its favicon accordingly. And with this listener, we can update the extension theme by reacting to a webpage reacting to a theme change :P . 

I actually looked into this last time in #14 , but did not add these findings in there... I'd appreciate that info to be preserved for the future - if you're okay with adding it as a comment. 

I think it's relevant because tabs.onUpdated listeners fire a lot, and it'd be great to get rid of them whenever we can.

Reply:
>  For example, github detects theme change and updates its favicon accordingly

It does this when you navigate onto its tab. And `chrome.windows.onFocusChanged` does the pretty same thing. So to dynamically change the pane theme, we just need to call a `aside-script.js` update theme fucntion from `background.js`. It isn't hard to implement. But maybe next time :)

Reply:
> It does this when you navigate onto its tab

I could have sworn it did not, and when I checked and changed my theme : 1 on my 3 open github tab instantly updated favicons... Weird ?! 
But then that means we can get rid of that onUpdated listener, doesn't it :D ?


> we just need to call a aside-script.js update theme fucntion from background.js

But that would need a aside-script.js already injected in some page and ready to be used, wouldn't it ? I'm not sure we can rely on having one lying around : the aside-script.js is inserted only when the user clicks on the extension icon, which he may not have done before changing his theme

Reply:
> But then that means we can get rid of that onUpdated listener, doesn't it :D ?

Fair enough 

> But that would need a aside-script.js already injected in some page and ready to be used, wouldn't it ?

No, theme is applied to the pane every time it opens (on [line 65](https://github.com/XFox111/ChromiumTabsAside/pull/18/files/fca6e18ec40c8b3e4bd11b3175ef54229bdf6fa9#diff-e0b683cb9b994def4e51e11a9e203afeR65)). So if you change theme and reopen the pane, it will change its color without page reloading

Reply:
Yes, for the pane that's right, but I was actually thinking about the browser icon : 
![image](https://user-images.githubusercontent.com/15179425/86412179-b5e95f80-bcbe-11ea-8ac1-bac4d20a8b71.png)
![image](https://user-images.githubusercontent.com/15179425/86412174-b41f9c00-bcbe-11ea-9d73-407cc371b138.png)

It becomes basically invisible if the theme is not synced, so I consider that more important than the pane theme

Reply:
Oh, I see. Well, there's no way to make it change instantly (I've found a SO thread where it has been written. Can't find now tho). The earliest moment it can be changed is when user switches tabs or navigates somewhere else on the settings tab (and this is why we need `onUpdated`, I've just recalled :P)

Reply:
And with GitHub open too. I'm still not sure in which cases it updates instantly, but to capture these screenshots I had to close my GitHub PR tab or the icon would update instantly.

Anyway, hopefully some options to deal with theme changes will get added to the chrome extension APIs, as dark themes are becoming increasingly popular. I'll check if it was already requested on the chromium issue tracker or something

Reply:
I suggest you to create a separate issue for this case, but I  still see no way how to implement this